lib/Wizard.php

```diff
@@ -685,6 +685,17 @@ public function guessBaseDN() {
 			}
 		}
 
+		//check whether the agent name is in email form
+		$i = \stripos($this->configuration->ldapAgentName, '@');
```

Comment:
`strrpos` should work better. We probably should get the position of the last "@" char in case there are multiple

Reply:
good idea. checking the last never hurts!